static/js/components/dialogs/EditVideoFormDialog.js

```diff
@@ -255,6 +260,16 @@ class EditVideoFormDialog extends React.Component<*, void> {
               validationMessage={errors ? errors.view_lists : ""}
             />
           </Radio>
+          <Radio
+            id="view-only-logged-in-users"
```

Comment:
On the collection form, the id is `view-logged-in-only` - can you make them the same on both (either one is fine)?